src/DocumentFormat.OpenXml.Framework/OpenXmlElement.cs

```diff
@@ -280,7 +280,7 @@ public bool HasAttributes
         /// <summary>
         /// Gets all extended attributes (attributes not defined in the schema) of the current element.
         /// </summary>
-        public IEnumerable<OpenXmlAttribute> ExtendedAttributes
+        public IReadOnlyList<OpenXmlAttribute> ExtendedAttributes
```

Comment:
Can't do this for two reasons:

1. Not supported in some targets (.net 3.5/4.0) that we support
2. (techincally for v3 we could but) it's a public API

Reply:
feel free to expose some of these internally as lists